### ISSUE TITLE
TBC PvP trinket

### ISSUE THREAD

**Issue by XyzKangUI** [Rhykes/GladiatorlosSA2#6]
Can you add UNIT_SPELLCAST_SUCCEEDED event for trinket alerting, since they apparently don't show in combat log. 

Example:

```
function GladiatorlosSA:UNIT_SPELLCAST_SUCCEEDED(event, unitTarget, castGUID, spellID)
	if gsadb.class and spellID == 42292 then
		local guid = UnitGUID(unitTarget)
		local c = self:ArenaClass(guid)
		if c then
			self:PlaySound(c);
		end
	end
end
```

Anyway, I hope this example will be sufficient for a TBC trinket alert implementation. Thanks for your work. 

On a side note, the drinking tooltip claims that "Alert Drinking" only announces when enemy is drinking. But it also announces friendly drinks. I suggest editing tooltip or `if UnitIsEnemy("player", uid) and gsadb.drinking then` ?

**Comment by Rhykes** [comment]
Wow I am so sorry, I did not get any notifications for this ticket.

Does that solution actually work? I was under the impression that I could only actually pull data like that from the combat log. If that's all it takes to fix the issue then I feel like a complete moron.

**Comment by XyzKangUI** [comment]
> Wow I am so sorry, I did not get any notifications for this ticket.
> 
> Does that solution actually work? I was under the impression that I could only actually pull data like that from the combat log. If that's all it takes to fix the issue then I feel like a complete moron.

Yes it works in TBCC, been using it for one month now.

**Comment by Rhykes** [comment]
That's great to hear, thank you for the fix! Will look at implementing them when I get time to sit down and do some coding. Will be updating the TBC-specific version with these since I guess there's still some outstanding issues with the Shadowlands/TBC cross-compatibility.

I can't believe it was that simple, I guess I still have a lot to learn.

**Comment by Rhykes** [comment]
Posted an update with both the Trinket and Drinking fixes to Curse, please give TBC-1.2 a try whenever Curse gets around to approving it.

Thank you very much for your help, going to close this ticket now. Feel free to open another if new issues crop up or something went horribly wrong with that update. (I don't play TBC so I wasn't able to extensively test them myself.)
